modules/nw-metallb-when-metallb.adoc

```diff
@@ -10,7 +10,9 @@ Using MetalLB is valuable when you have a bare-metal cluster, or an infrastructu
 
 You must configure your networking infrastructure to ensure that network traffic for the external IP address is routed from clients to the host network for the cluster.
 
-After deploying MetalLB with the MetalLB Operator, when you add a service of type `LoadBalancer`, MetalLB provides a platform-native load balancer.
+After deploying MetalLB with the MetalLB Operator, when you add a service of type `LoadBalancer`, MetalLB provides a platform-native load balancer. 
+
+When external traffic enters your {product-title} cluster through a MetalLB LoadBalancer service, the return traffic to the client will have the LoadBalancer's external IP address as the source IP.
```

Comment:
```suggestion
When external traffic enters your {product-title} cluster through a MetalLB `LoadBalancer` service, the return traffic to the client has the external IP address of the load balancer as the source IP.
```
See the ISG on [verb tense](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) and the docs guidelines on [general](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) vs specific [API object references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references).

Also, the ISG says [to avoid using possessives](https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives) with inanimate objects.